src/ape_pm/_cli.py

```diff
@@ -148,6 +149,67 @@ def install(cli_ctx, package, name, version, ref, force):
         cli_ctx.logger.success("All project packages installed.")
 
 
+@cli.command()
+@ape_cli_context()
+@click.argument("package", nargs=1, required=True)
+@click.argument("versions", nargs=-1, required=False)
+@click.option(
+    "-y", "--yes", is_flag=True, help="Automatically confirm the removal of the package(s)"
+)
+def remove(cli_ctx, package, versions, yes):
+    """
+    Remove a package.
+
+    This command removes a package from the installed packages.
+
+    If specific versions are provided, only those versions of the package will be removed.
+    If no versions are provided, the command will prompt you to choose versions to remove.
+    You can also choose to remove all versions of the package.
+
+    Examples:
+    - \nRemove specific versions: ape pm remove OpenZeppelin 4.9.0 4.9.3
```

Comment:
this part is not rendering that nicely in my terminal.

try

```
ape pm remove --help
```

this is what it looks like for me:

```
      Examples:     -  Remove specific versions: ape pm remove OpenZeppelin
      4.9.0 4.9.3     -  Prompt to choose versions: ape pm remove OpenZeppelin
      -  Remove all versions: ape pm remove OpenZeppelin -y
```

can we make this look nicer?

Reply:
yup, I have updated it locally. I will push the changes